discord/guild.go

```diff
@@ -4,8 +4,8 @@ import "time"
 
 // https://discord.com/developers/docs/resources/guild#guild-object
 type Guild struct {
-	// ID is the guild id.
-	ID GuildID `json:"id"`
+	// VanityURL is the the vanity url code for the guild.
```

Comment:
While not related to this change, this should be updated to follow the godoc format